backends/ref/ceed-ref-restriction.c

```diff
@@ -26,7 +26,7 @@ static int CeedElemRestrictionApply_Ref(CeedElemRestriction r,
   int ierr;
   const CeedScalar *uu;
   CeedScalar *vv;
-  CeedInt esize = r->nelem*r->elemsize;
+  CeedInt esize = r->nelem*r->elemsize*ncomp;
```

Comment:
@jedbrown : Is this change okay? We should take into account number of
components as well right?

Reply:
Nevermind, I fix this in eb5d317